wiki/Article_Styling_Criteria/en.md

```diff
@@ -439,46 +746,115 @@ Other brandings that aren't covered by the osu! official branding must use space
 - `osu! client`
 - `osu! wiki`
 
+---
+
+<!-- Ephemeral ruled to use a full stop. in:osu-wiki during:2018-05-30 from:ephemeralis#0001 stop -->
+
+Try to avoid letting the word `osu!` end the sentence. If you are unable to rewrite it, then use a full stop (`.`) (or question mark (`?`), if appropriate) to get `osu!.` (or `osu!?`). For example:
+
+```
+Avoid: peppy created osu!.
+
+Do:    osu! was created by peppy.
+```
+
+Never end a sentence with `osu!` and have it followed by an exclamation mark (`!`) (never use `osu!!`). You will need to restructure the sentence or the pargraph to ensure that this will never occur.
+
 ### Capitalisation
 
 #### Article names
 
-When writing about a topic that has an article, **never capitalise it unless it is a link** (with exceptions, e.g. BanchoBot). For example:
+When writing about a topic that has an article, never capitalise it unless it is a link (with exceptions, e.g. BanchoBot). For example:
 
-- `The [Chat Console](/wiki/Chat_Console) is where you go to chat, using IRC (Internet Relay Chat). The chat console is located in the game client.`
+```md
+The [Chat Console](/wiki/Chat_Console) is where you go to chat, using IRC (Internet Relay Chat). The chat console is located in the game client.
+```
 
 #### Game modifiers
 
-**Game modifiers must be capitalised.** For example:
+Game modifiers should have the word `mod` after the mod name to reduce ambiguity. These must be spelt as seen below (note the letter casing and spacing):
+
+- `Easy` or `EZ`
+- `No Fail` or `NF`
+- `Half Time` or `HT`
+- `Daycore` or `DC`
+- `Hard Rock` or `HR`
+- `Sudden Death` or `SD`
+- `Perfect` or `PF`
+- `Double Time` or `DT`
+- `Nightcore` or `NC`
+- `Fade In` or `FI`
+- `Hidden` or `HD`
+- `Flashlight` or `FL`
+- `Relax` or `RL`
+- `Autopilot` or `AP`
+- `Target Practice` or `TP`
+- `Spun Out` or `SO`
+- `1K`, `2K`, `3K`, `4K`, `5K`, `6K`, `7K`, `8K`, and `9K`
+  - If referring to the key mods collectively, use `xK`.
+- `Co-op` or `CO`
+- `Random` or `RD`
+- `Auto` or `AT`
+- `Cinema` or `CM`
+- `Touch Device` or `TD`
+
+---
+
+Tournament articles are an exception to this and can use camel case (omit the space but keep their letter casing as seen above). For mappools with no or free mods, the game modifier name must be spelt as seen below:
 
-- `Hard Rock`
-- `Nightcore`
-- `Double Time`
-- `Easy`
+- `No Mods`, `No Mod`, `NoMods`, `NoMod`, or `NM`
+- `Free Mods`, `Free Mod`, `FreeMods`, `FreeMod`, or `FM`
 
```

Comment:
Can you add the `Tiebreaker` bracket. Not really a mod name, so idk where to put it.

Don't know the format of `Tiebreaker`, whether it is `TieBreaker` or not. Googled it and it should be `Tiebreaker`.